Userland/Applications/TextEditor/MainWidget.cpp

```diff
@@ -366,6 +366,7 @@ void MainWidget::initialize_menubar(GUI::Window& window)
     edit_menu.add_action(m_editor->copy_action());
     edit_menu.add_action(m_editor->paste_action());
     edit_menu.add_separator();
+    edit_menu.add_action(m_editor->insert_emoji_action());
```

Comment:
I think this should be in a separate `TextEditor:` commit (and maybe we can find a few more places to enable it?)